lib/beaker-hostgenerator/cli.rb

```diff
@@ -153,16 +152,14 @@ def print_platforms_and_roles
       puts "   6432 => 64-bit OS with 32-bit Puppet (Windows Only)"
       puts "\n"
 
-      roles = BeakerHostGenerator::Utils.get_roles
       puts "valid beaker-hostgenerator host roles:  "
-      roles.each do |k,v|
+      ROLES.each do |k,v|
```

Comment:
I wanted to get rid of the Utils class as most of the methods made more sense in the new Data class, and this `get_roles` method wasn't really necessary as the usages of it were safe to replace with direct constant access.